scripts/asm-installer/release_asm_installer

```diff
@@ -118,6 +127,28 @@ get_stable_version() {
   echo "${VER//+/-}"
 }
 
+get_version_file_and_lock() {
+  local FILE_PATH; FILE_PATH=${BUCKET_PATH}/${STABLE_VERSION_FILE}
+
+  if [[ ! $(curl -O "${BUCKET_URL}/${FILE_PATH}") ]]; then
+    echo "Create New Version File ${STABLE_VERSION_FILE}"
+    true >| ${STABLE_VERSION_FILE}
+    gsutil cp "${STABLE_VERSION_FILE}" gs://"${FILE_PATH}"
+  fi
```

Comment:
A few things here.

High level, I'm not sure we should do this in the script. I can't imagine why we'd ever delete this file once we have it, and if we do this then it's very possible that whoever is running the script might not even notice the log line saying that we're creating a new one. Unless we have some known situations where this might happen, I'd rather fail loudly so that it's extremely clear what's going on.

Style issues:

1. if you're checking the return value of a function you don't need [[]] `if ! curl -O ... ;then`
2. generally, if you're echoing something that you intend a human to read, default to putting it in `stderr` with >&2
3. to create a file, I generally see `touch $FILE` not `true > $FILE`.

Reply:
Thanks for the advice! As for 3, I use `true >| $FILE` because the `curl -O ...` will create that file, with the xml error message inside as the file content if the curl failed. Therefore, I meant to override/truncate that file.

Reply:
Good point on `true >| $file`, I didn't think about that.